src/components/main/NoDataFound.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const NoDataFound = () => {

```

Comment:
Maybe when display the noDataFound message, you could clear out the feature movie and just show the error so the user can see the message right away. Also when the user is using the search bar to look for a movie, perhaps remove the feature movie and only showcase the ones user is searching for will make it a better user experience.

Reply:
Thank you for your suggestion. My aim was to implement the same functionality you suggested. However, I was not able to do it because of passing b/w comp. that are reusable in both pages. but I just figured that out. Thank you for this suggestion, otherwise i was gonna push this one in future goals.